examples/webpack.config.dev.js

```diff
@@ -0,0 +1,74 @@
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const path = require('path');
+
+const babelLoaderOptions = {
```

Comment:
Please import already existing settings from `babel.config.js` or just let the loader use the file itself.